### ISSUE TITLE
Change how onPause is handled to better support multiwindow mode on Android

### ISSUE THREAD

**Issue by GrantOjanen** [libgdx/libgdx#5728]
#### Issue details
On Android 7 when they added multi-window support `onPause` is called when the activity no longer has focus, but is still visible. `onStop` is called when the activity is no longer visible. So if the window is visible, but doesn't have focus the game should continue to render new frames. If the user resizes the window while it doesn't have focus the game needs to render a new frame to adjust to the new size.

The game should be notified that it no longer has focus (`onPause`,) so the developer can display a pause menu. The game should stop rendering new frames when it's no longer visible (`onStop`.) Vise versa for `onResume` and `onStart`.

#### Version of LibGDX and/or relevant dependencies
libGDX v1.9.9

#### Please select the affected platforms
- [x] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by MrStahlfelge** [comment]
I think this is controversial. You can argue that the game should stop render if it has not the focus, too. It feels like a breaking change for a feature probably not used very often.

**Comment by MobiDevelop** [comment]
I'd think it would have to be opt-in.

**Comment by MrStahlfelge** [comment]
Yes, with a lot of work this can be done non-breaking. All the code is just tied to onPause() at the moment, so changes here are highly likely to break a thing.
